src/components/molecules/Chatbox/useTriggerScrollFix.tsx

```diff
@@ -1,18 +1,18 @@
-import { WithId } from "../../../utils/id";
-import { MessageToDisplay } from "../../../types/chat";
+import { WithId } from "utils/id";
+import { MessageToDisplay } from "types/chat";
 import { useEffect, useRef } from "react";
```

Comment:
I know this is OOS, but can you please rearrange these imports.

Reply:
@denisdimitrov I'm working on the automatic ordering now, so it's okay to leave it as is for now